src/components/JobBuild.jsx

```diff
@@ -18,12 +18,27 @@ class JobBuild extends Component {
             </div>
         );
     }
+
+    buildResult() {
```

Comment:
I'm planing on moving a lot of the actual Mozaïk components to simple functions instead of classes, having no context could help, so perhaps this utility should be outside the component, for example:

```javascript
const JENKINS_BUILD_STATUS_BUILDING = 'BUILDING';
const JENKINS_BUILD_STATUS_UNKNOWN  = 'UNKNOWN';
// perhaps other statuses could be added as constants

/**
 * Returns a build status from given build info.
 *
 * @param {object} build - the jenkins build object
 * @returns {string}
 */
const getBuildStatus = (build) => {
    if (build.result) {
        return build.result;
    }

    return build.building ? JENKINS_BUILD_STATUS_BUILDING : JENKINS_BUILD_STATUS_UNKNOWN;
};
```